commands/checker.js

```diff
@@ -67,42 +78,55 @@ export const checker = async ({ json, recreateCheckerIdOnError, experimental })
     await fs.mkdir(join(paths.runtimeState, runtimeName), { recursive: true })
   }
 
-  activities.onActivity(activity => {
+  activities.onActivity((activity) => {
     if (json) {
-      console.log(JSON.stringify({
-        type: `activity:${activity.type}`,
-        subnet: activity.source,
-        message: activity.message
-      }))
+      console.log(
+        JSON.stringify({
+          type: `activity:${activity.type}`,
+          subnet: activity.source,
+          message: activity.message,
+        }),
+      )
     } else {
       process.stdout.write(formatActivityObject(activity))
     }
   })
 
-  metrics.onUpdate(metrics => {
+  metrics.onUpdate((metrics) => {
     if (json) {
-      console.log(JSON.stringify({
-        type: 'jobs-completed',
-        total: metrics.totalJobsCompleted,
-        rewardsScheduledForAddress: formatEther(metrics.rewardsScheduledForAddress)
-      }))
+      console.log(
+        JSON.stringify({
+          type: 'jobs-completed',
+          total: metrics.totalJobsCompleted,
+          rewardsScheduledForAddress: formatEther(
+            metrics.rewardsScheduledForAddress,
+          ),
+        }),
+      )
     } else {
-      console.log(JSON.stringify({
-        totalJobsCompleted: metrics.totalJobsCompleted,
-        rewardsScheduledForAddress:
-          formatEther(metrics.rewardsScheduledForAddress)
-      }, null, 2))
+      console.log(
+        JSON.stringify(
+          {
+            totalJobsCompleted: metrics.totalJobsCompleted,
+            rewardsScheduledForAddress: formatEther(
+              metrics.rewardsScheduledForAddress,
+            ),
+          },
+          null,
+          2,
+        ),
+      )
```

Comment:
This would be better formatted if we used `%o` instead of JSON.stringify, I think.

```js
console.log('%o', {
 totalJobsCompleted: metrics.totalJobsCompleted,
 rewardsScheduledForAddress: formatEther(
 metrics.rewardsScheduledForAddress,
})
```

I feel such a change is out of the scope of this PR. WDYT?